sdk/cosmosdb/cosmos/src/client/Container/ContainerRequest.ts

```diff
@@ -7,5 +7,11 @@ import { VerboseOmit } from "../../utils/types";
 export interface ContainerRequest extends VerboseOmit<ContainerDefinition, "partitionKey"> {
   /* Throughput for this container. */
   throughput?: number;
+  maxThroughput?: number;
```

Comment:
These inline docs need an update. The comments will show up in the reference docs https://docs.microsoft.com/en-us/javascript/api/@azure/cosmos/?view=azure-node-latest